IntegrationTest/UserEndpoints.cs

```diff
@@ -0,0 +1,81 @@
+using System.Net;
+using API;
+using Core.Shared;
+using Infrastructure.Authentication.Contracts;
+using Infrastructure.Authentication.Models;
+using IntegrationTest.Setup;
+using Microsoft.Extensions.DependencyInjection;
+using NSubstitute;
+using Xunit.Abstractions;
+
+namespace IntegrationTest;
+
+[Collection(CollectionDefinitions.NonParallelCollectionName)]
+public class UserEndpoints : IClassFixture<TestWebApplicationFactory<Program>>
+{
+	private readonly TestWebApplicationFactory<Program> _factory;
+	private readonly HttpClient _client;
+
+	public UserEndpoints(TestWebApplicationFactory<Program> factory)
+	{
+		_factory = factory;
+		_client = factory.CreateClient();
+	}
+
+	[Fact]
+	public async Task GetAnonUser_ShouldReturn_Unauthorized()
+	{
+		var response = await _client.GetAsync("v1/users/1");
+
+		Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+	}
+	
+	[Fact]
+	public async Task GetAnonUser_ShouldReturn_NotFoundSessionIdCantBeFound()
+	{
+		using var scope = _factory.Services.CreateScope();
+		var userRepo = scope.ServiceProvider.GetService<IUserRepository>();
+		userRepo!.GetAnonUserSessionByIdAsync(Arg.Any<int>()).Returns(0);
+		_client.AddAnonAuth(1,1);
```

Comment:
What is the difference to this Auth, and the other method used throughout the integration tests?